Project/ISVR/src/jeet201801232.py

```diff
@@ -0,0 +1,197 @@
+import psutil
+import os
+import webbrowser
+import random
+import string
+from sys import platform
+import psutil as p
+from va import speak,ask
+
+def show_downloads():

```

Comment:
This is not generalized, make sure it works for all computers see: https://stackoverflow.com/questions/35851281/python-finding-the-users-downloads-folder
https://askubuntu.com/questions/137896/how-to-get-the-user-downloads-folder-location-with-python
of if needed just ask the user.